supabase/templates/recovery.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<html>
+    <head>
+        <style>
+            body {
+                font-family: Arial, sans-serif;
+                background-color: #f4f4f4;
+                margin: 0;
+                padding: 0;
+            }
+            .container {
+                width: 100%;
+                max-width: 600px;
+                margin: 0 auto;
+                background-color: #ffffff;
+                padding: 20px;
+                border-radius: 8px;
+                box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+            }
+            .header {
+                text-align: center;
+                padding: 10px 0;
+            }
+            .header h2 {
+                margin: 0;
+                color: #333333;
+            }
+            .content {
+                padding: 20px;
+                line-height: 1.6;
+                color: #555555;
+            }
+            .content p {
+                margin: 0 0 10px;
+            }
+            .button {
+                display: block;
+                width: 200px;
+                margin: 20px auto;
+                padding: 10px;
+                text-align: center;
+                background-color: #007bff;
+                color: #ffffff;
+                text-decoration: none;
+                border-radius: 5px;
+            }
+            .footer {
+                text-align: center;
+                padding: 10px 0;
+                font-size: 12px;
+                color: #777777;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="container">
+            <div class="header">
+                <h2>Reset Password</h2>
+            </div>
+            <div class="content">
+                <p>Hello,</p>
+                <p>To reset your password, follow the link below:</p>
+                <a href="{{ .ConfirmationURL }}/auth-callback" class="button"
+                    >Reset my password</a
+                >
+                <p>
+                    <strong>Warning:</strong> If the password reset request did
+                    not originate from you, please contact us immediately as
+                    this may be a fraudulent attempt.
```

Comment:
thought (non-blocking): it could be nice to be more explicit about how the users are supposed to contact us. Dedicated email? GH Issue? Or maybe remove this whole sentence as it is not really applicable at the moment.